packages/frontend/src/components/Pagination/components/Card/Card.tsx

```diff
@@ -15,23 +15,19 @@ export const Card: React.FC<CardProps> = ({ id, photoUrl, title, rating, total,
     setIsFilled(!isFilled);
   };
 
-  const isRatingAndTotal = !!rating && !!total;
+  const hasRatingAndTotal = !!rating && !!total;
 
   return (
     <Wrapper to={`${Routes.Recipe}${id}`}>
       <ImageContainer bgImage={photoUrl}></ImageContainer>
       <HeartIcon src={heartIcon} $isFilled={isFilled} onClick={toggleFill} />
       <RecipeName>{title}</RecipeName>
-      <BottomPanel>
-        {isRatingAndTotal ? (
-          <>
-            <Rating rating={rating} total={total} />
-            <Author>{author}</Author>
-          </>
-        ) : (
-          <Author>{author}</Author>
-        )}
-      </BottomPanel>
+      {hasRatingAndTotal || author ? (
```

Comment:
Autor będzie zawsze, więc nie rozumiem tego sprawdzania. Niżej w 27 linii wystarczy sprawdzenie, a to w 28 jest nie potrzebne. Do BottomPanel możesz podać propsa, który w którym sprawdzasz czy jest rating i na tej podstawie dać space-between, albo right.
Jeżeli nie będzie raitingu to autor będzie przylepiony poprawnie do lewej.